cmd/clusterctl/client/config.go

```diff
@@ -76,10 +77,10 @@ type GetClusterTemplateOptions struct {
 	KubernetesVersion string
 
 	// ControlPlaneMachineCount defines the number of control plane machines to be added to the workload cluster.
```

Comment:
We should update this comment giving users directions on how to set these values with environment variables, and what's the default in case they aren't set